addons/mail/static/src/models/popover_view/popover_view.js

```diff
@@ -17,13 +17,81 @@ registerModel({
         },
     },
     recordMethods: {
+        /**
+         * Returns whether the given html element is inside the component
+         * of this popover view.
+         *
+         * @param {Element} element
+         * @returns {boolean}
+         */
+        contains(element) {
+            return Boolean(this.component && this.component.el.contains(element));
```

Comment:
actually component.el will be obsolete with Owl 2, this should directly use root.el